src/py42/clients/archive.py

```diff
@@ -120,6 +120,7 @@ def stream_to_device(
         archive_password=None,
         encryption_key=None,
         show_deleted=None,
+        file_location=None,
```

Comment:
Should this actually be exposed on the public client method?

Reply:
I don't really see why.  Like, there are only two options for the API parameter:

`ORIGINAL_LOCAITION`: for the restoring to the original location (the one to satisfy the acceptance criteria) and 

`TARGET_LOCATION`: which appears to be the default behavior and requires the `restore_path` parameter.  

When using both `restore_path` and `file_location=ORIGINAL_LOCATION`, it does not honor `restore_path` because you are telling it to restore the files to the original location.

Therefore, it would not make sense that `restore_path` is a required parameter - yet it is.

Reply:
But, we do need to know either the restore path or if they just want to use the `original location` strategy.

`restore_path=ORIGINAL_LOCTION` would suffice for me but at minimum, `restore_path` should not be a required parameter but it would one or the other (either provide the `file_location` param or the `restore_path` param)

Reply:
Haha no